lib/plugins/personalization.js

```diff
@@ -12,11 +12,11 @@ var _ = require('../utils/lodash')
  *    - defaults to 'cdp.in.treasuredata.com'
  *
  *    Possible values:
- *    Region    cdpHost                       host
- *    AWS East  cdp.in.treasuredata.com       in.treasuredata.com
- *    AWS Tokyo cdp-tokyo.in.treasuredata.com tokyo.in.treasuredata.com
- *    AWS EU    cdp-eu01.in.treasuredata.com  eu01.in.treasuredata.com
- *    IDCF      cdp-idcf.in.treasuredata.com  idcf.in.treasuredata.com
+ *    Region     cdpHost                       host
+ *    AWS East   cdp.in.treasuredata.com       in.treasuredata.com
+ *    AWS Tokyo  cdp-tokyo.in.treasuredata.com tokyo.in.treasuredata.com
+ *    AWS EU     cdp-eu01.in.treasuredata.com  eu01.in.treasuredata.com
+ *    AWS Korea  cdp-ap02.in.treasturedata.com ap02.in.treasuredata.com
```

Comment:
We should use official name: `AWS Asia Pacific (Seoul)`

Reply:
👍